extensions/clusters/gke/gke_cluster_config.go

```diff
@@ -143,10 +143,14 @@ func clusterAddonsBuilder(clusterAddons *ClusterAddons) *management.GKEClusterAd
 }
 
 func cmekConfigBuilder(cmekConfig *CMEKConfig) *management.CMEKConfig {
-	return &management.CMEKConfig{
-		KeyName:  cmekConfig.KeyName,
-		RingName: cmekConfig.RingName,
+	var CMEKConfig *management.CMEKConfig
+	if cmekConfig != nil {
```

Comment:
Is there a chance that this is a nil? Is this something new?

Reply:
Yes, this change allows to omit specifying _customerManagedEncryptionKey_ in CATTLE_TEST_CONFIG's gkeClusterConfig  section.